packages/generator/src/generation/code-generation/ts-helper.ts

```diff
@@ -0,0 +1,109 @@
+import ts, {factory} from "typescript"
+
+class TypescriptHelper {
+    
+    createImportDeclaration(importSpecifier: string, module:string): ts.ImportDeclaration {
+        return factory.createImportDeclaration(
+            undefined,
+            undefined,
+            factory.createImportClause(
+              false,
+              undefined,
+              factory.createNamedImports([
+                factory.createImportSpecifier(
+                  undefined,
+                  factory.createIdentifier(importSpecifier)
+                ),
+              ])
+            ),
+            factory.createStringLiteral(module)
+          )
+    }
+
+    createNameSpaceImport(namespace: string, module: string): ts.ImportDeclaration {
+      return factory.createImportDeclaration(
+        undefined,
+        undefined,
+        factory.createImportClause(
+          false,
+          undefined,
+          factory.createNamespaceImport(factory.createIdentifier(namespace))
+        ),
+        factory.createStringLiteral(module)
+      )
+    }
+
+    //TODO: improve this to be able also process namespace imports
+    uniqueImports(imports:ts.ImportDeclaration[]): ts.ImportDeclaration[]{
```

Comment:
move this function please in new file: imports/unique.ts

Simmilar to what we have in https://github.com/iteria-app/lowcode/commit/3a7d7b4525b4c71ea79943c6787b2aae2a74198f#diff-02b50984ce1bcacda4cdc94fdf741c13689ed25757b4e6415bab8b6f83c56432